src/replica/bulk_load/replica_bulk_loader.cpp

```diff
@@ -409,6 +412,10 @@ error_code replica_bulk_loader::download_sst_files(const std::string &app_name,
                 uint64_t f_size = 0;
                 error_code ec = _stub->_block_service_manager.download_file(
                     remote_dir, local_dir, f_meta.name, fs, f_size);
+                if (ec == ERR_OK && f_size == 0) {
```

Comment:
It's a bit of confused, how about define a new code like `ERR_DOWNLOADED`?

Reply:
I remove this code, which is not related with this pr, I will add another pr to solve it.